Activities/Cryptography/UiPath.Cryptography.Activities/UiPath.Cryptography.Activities.csproj

```
@@ -6,10 +6,14 @@
 		<OutputPath>..\..\Output\Activities\Cryptography\</OutputPath>
 	</PropertyGroup>
 	<ItemGroup>
+		<PackageReference Include="UiPath.Telemetry.Client" />
 		<PackageReference Include="UiPath.Workflow.Runtime" />
 		<PackageReference Include="UiPath.Workflow" />
 		<PackageReference Include="System.Activities.Metadata" />
 		<PackageReference Include="System.Activities.ViewModels" />
+		<PackageReference Include="UiPath.Activities.Contracts" PrivateAssets="All" Version="24.10.1" />
```

Version should be in Directory.build.targets

```suggestion
		<PackageReference Include="UiPath.Activities.Contracts" PrivateAssets="All" />
```